### ISSUE TITLE
Goal statement

### ISSUE THREAD

**Issue by Maryamvalian** [KarenKarenWang/cas741_project1#5]
@KarenKarenWang I might be wrong, but I think G3 is not verifiable or may be hard to verify.

**Comment by joedf** [comment]
Yes, I agree that it could be difficult to verify. Perhaps, this can be rectified by saying:
> This model will output **an estimate** of CO<sub>2</sub> emissions [...]

And maybe provide a source of the approximate emission rates. So finally, it could be:
> This model will output **an estimate** of CO<sub>2</sub> emissions **based on approximate emission rates (source here)** [...]

**Comment by KarenKarenWang** [comment]
fixed, close for now
